### DISCUSSION TITLE
How to delete old target file each time before generating new allure-results?

### DISCUSSION THREAD

**Discussion by AndjelaPop** [allure-framework/allure2#1853] [Questions & Support]
When I run the new test, target file is automatically adding a new allure-results data. How can I avoid having some old report data in Allure reports? When I don't change the data in my API tests, the report is good, just showing history data, but when I change some data in the tests, the report looks pretty confusing.
Does someone have the same issue? Or know which commad is removing target file before each run?

**Comment by cheshi-mantu** [comment]
hi @AndjelaPop 
could you please shed the light here?
how do you execute your tests?
